src/com_tjnotifications/admin/views/notifications/tmpl/default.php

```diff
@@ -99,6 +100,12 @@
 						<th width="10%" class="hidden-phone">
 							<?php echo HTMLHelper::_('grid.sort', Text::_("COM_TJNOTIFICATIONS_VIEW_NOTIFICATIONS_DEFAULT_FIELD_KEY"), 'key', $listDirn, $listOrder);?>
 						</th>
+						<th width="10%" class="hidden-phone">
```

Comment:
these columns are not sortable

Reply:
resolved except language,email status and Unsubscribed Users all are performing sorting operation.